### DISCUSSION TITLE
LNM Flight plan does not match simbrief imported plan using Simbrief in MSFS

### DISCUSSION THREAD

**Discussion by stevebarker12** [albar965/littlenavmap#1003] [General]
When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft  and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)

Its not far off but not exactly the same.
Just wondered if i have done something wrong or this is expected.
I notice AIRAC cycle 2209 is permanently  displayed in  scenery library menu but i have loaded (I think !)  latest up to date cycle into  LNM
I use same navigraph updater software for LNM and MSFS.
Thankyou for any comments

**Comment by albar965** [comment]
> When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)

Please post the whole content of the LNM SimBrief import window for such a failing plan. No need for a screenshot. You can right click into the result message, select all and copy.

> When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)
> 
> Its not far off but not exactly the same. Just wondered if i have done something wrong or this is expected. I notice AIRAC cycle 2209 is permanently displayed in scenery library menu but i have loaded (I think !) latest up to date cycle into LNM I use same navigraph updater software for LNM and MSFS. Thankyou for any comments

The Navigraph FMS Data Manager sometimes installs the files into the wrong folder. See here: https://albar965.github.io/littlenavmap_navigraph.html
You can be sure the import worked once LNM shows the right cycle in the menu or window title bar. Close LNM before installing the update.

Also keep this in mind: https://www.littlenavmap.org/manuals/littlenavmap/release/2.8/en/SCENERY.html#load-scenery-library-dialog-msfs-apt-navdata

I rather suspect that the different cycles in LNM and MSFS cause the issues.

Alex

**Comment by stevebarker12** [comment]
Thanks very much.Ill go through your comments - my initial thought is i must not be importing the plan into the right place if an old cycle - 2209- is still being displayed.Ill let you know what the fault was when ive been through your suggestions.Kind regardsSteveOn 2 Mar 2023 16:02, Alexander Barthel ***@***.***> wrote:

When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)

Please post the whole content of the LNM SimBrief import window for such a failing plan. No need for a screenshot. You can right click into the result message, select all and copy.

When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)
Its not far off but not exactly the same. Just wondered if i have done something wrong or this is expected. I notice AIRAC cycle 2209 is permanently displayed in scenery library menu but i have loaded (I think !) latest up to date cycle into LNM I use same navigraph updater software for LNM and MSFS. Thankyou for any comments

The Navigraph FMS Data Manager sometimes installs the files into the wrong folder. See here: https://albar965.github.io/littlenavmap_navigraph.html
You can be sure the import worked once LNM shows the right cycle in the menu or window title bar. Close LNM before installing the update.
Also keep this in mind: https://www.littlenavmap.org/manuals/littlenavmap/release/2.8/en/SCENERY.html#load-scenery-library-dialog-msfs-apt-navdata
I rather suspect that the different cycles in LNM and MSFS cause the issues.
Alex

**Comment by stevebarker12** [comment]
I was updating Airac navigraph to wrong  LNM folderProblem sortedThankyou for your instructions which sorted the problem.Kind regardsSteveOn 2 Mar 2023 16:21, Steve Barker ***@***.***> wrote:Thanks very much.Ill go through your comments - my initial thought is i must not be importing the plan into the right place if an old cycle - 2209- is still being displayed.Ill let you know what the fault was when ive been through your suggestions.Kind regardsSteveOn 2 Mar 2023 16:02, Alexander Barthel ***@***.***> wrote:

When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)

Please post the whole content of the LNM SimBrief import window for such a failing plan. No need for a screenshot. You can right click into the result message, select all and copy.

When I import a Simbrief flight plan into LNM then the purple route displayed in LNM does not match the purple route in the aircraft and aircraft follows simbrief route (I use MSFS 2020 and import latest same database from navigraph into LNM and the aircraft eg Fenix A320 . I load scenery library in LNM as well.)
Its not far off but not exactly the same. Just wondered if i have done something wrong or this is expected. I notice AIRAC cycle 2209 is permanently displayed in scenery library menu but i have loaded (I think !) latest up to date cycle into LNM I use same navigraph updater software for LNM and MSFS. Thankyou for any comments

The Navigraph FMS Data Manager sometimes installs the files into the wrong folder. See here: https://albar965.github.io/littlenavmap_navigraph.html
You can be sure the import worked once LNM shows the right cycle in the menu or window title bar. Close LNM before installing the update.
Also keep this in mind: https://www.littlenavmap.org/manuals/littlenavmap/release/2.8/en/SCENERY.html#load-scenery-library-dialog-msfs-apt-navdata
I rather suspect that the different cycles in LNM and MSFS cause the issues.
Alex

**Reply by albar965** [reply]
Great. Let me know if you still see issues with SimBrief import.

**Comment by stevebarker12** [comment]
Not at moment but thanksSteveSent from my Galaxy
-------- Original message --------From: Alexander Barthel ***@***.***> Date: 03/03/2023  11:47  (GMT+00:00) To: albar965/littlenavmap ***@***.***> Cc: stevebarker12 ***@***.***>, Author ***@***.***> Subject: Re: [albar965/littlenavmap] LNM Flight plan does not match simbrief imported plan using Simbrief in MSFS (Discussion #1003) 
Great. Let me know if you still see issues with SimBrief import.
